src/server/unittest.c

```diff
@@ -65,6 +65,8 @@ main(int argc, char **argv)
 		exit(1);
 	}
 	dump_login(login);
+	// tds->conn->tds_version = 0x702;
+	// tds->conn->product_version = G.TDS_MS_VER(10, 0, 6000);
```

Comment:
also some systems not liking `//` style comments.

Reply:
changed to /* */ format

Reply:
I cannot see the change, maybe you forgot to push it upstream?